tests/conftest.py

```diff
@@ -178,3 +178,9 @@ def custom_template(tmp_dir, dvc):
 
 
 scriptify_fixture = pytest.fixture(lambda: scriptify, name="scriptify")
+
+
+@pytest.fixture(scope="session", autouse=True)
+def webbrowser_open(session_mocker):
+    session_mocker.patch("dvc.repo.live.webbrowser_open")
```

Comment:
Let's drop this? We may still want to test `webbrowser_open`?

Reply:
It is still possible to re-patch inside the test and it works as intended

Reply:
And without it, the global patch doesn't work for the test in `tests/func/live`

Reply:
I don't think it's a good idea to mock `dvc.repo.live.webbrowser_open` globally.

Reply:
Merging, let's see if we can fix this after https://github.com/iterative/dvc/pull/6993. We may be able to mock `ui.open_browser`.

Reply:
Okay, making the imports for `webbrowser` lazy makes it work. Pushing the changes.